src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dns/ClusterDnsCacheSpec.scala

```diff
@@ -1,142 +1,50 @@
 package org.broadinstitute.dsde.workbench.leonardo.dns
 
-import java.time.Instant
-import java.util.UUID
-
-import akka.actor.ActorSystem
 import akka.http.scaladsl.model.Uri.Host
-import akka.pattern.ask
-import akka.testkit.{TestActorRef, TestKit}
-import akka.util.Timeout
 import org.broadinstitute.dsde.workbench.leonardo.ClusterEnrichments.clusterEq
 import org.broadinstitute.dsde.workbench.leonardo.{CommonTestData, GcsPathUtils}
 import org.broadinstitute.dsde.workbench.leonardo.db.{DbSingleton, TestComponent}
 import org.broadinstitute.dsde.workbench.leonardo.dns.ClusterDnsCache._
 import org.broadinstitute.dsde.workbench.leonardo.model._
 import org.broadinstitute.dsde.workbench.leonardo.model.google._
-import org.broadinstitute.dsde.workbench.model.google.GcsBucketName
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
 import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
 
-import scala.concurrent.duration._
-
 /**
   * Created by rtitle on 9/1/17.
   */
-class ClusterDnsCacheSpec extends TestKit(ActorSystem("leonardotest")) with FlatSpecLike with Matchers with BeforeAndAfterAll with TestComponent with ScalaFutures with Eventually with CommonTestData with GcsPathUtils {
-
-  implicit val timeout = Timeout(5 seconds)
+class ClusterDnsCacheSpec extends FlatSpecLike with BeforeAndAfterAll with TestComponent with ScalaFutures with Eventually with CommonTestData {
 
   override def afterAll(): Unit = {
-    TestKit.shutdownActorSystem(system)
     super.afterAll()
   }
 
-  val cluster1 = makeCluster(1).copy()
-
-  val cluster2 = makeCluster(2).copy(status = ClusterStatus.Creating,
-                                     dataprocInfo = makeDataprocInfo(2).copy(hostIp = None))
+  val cluster1: Cluster = makeCluster(1).copy()
 
-  val cluster3 = makeCluster(3).copy(status = ClusterStatus.Stopping,
-                                     dataprocInfo = makeDataprocInfo(3).copy(hostIp = None))
+  val cluster2: Cluster = makeCluster(2).copy(status = ClusterStatus.Creating,
+    dataprocInfo = makeDataprocInfo(2).copy(hostIp = None))
 
+  val clusterDnsCache = new ClusterDnsCache(proxyConfig, DbSingleton.ref, dnsCacheConfig)
   it should "update maps and return clusters" in isolatedDbTest {
-    val actorRef = TestActorRef[ClusterDnsCache](ClusterDnsCache.props(proxyConfig, DbSingleton.ref))
-
-    // init maps to empty
-    actorRef.underlyingActor.ProjectNameToHost = Map.empty
-    ClusterDnsCache.HostToIp = Map.empty
 
     // save the clusters to the db
     cluster1.save() shouldEqual cluster1
     cluster2.save() shouldEqual cluster2
-    cluster3.save() shouldEqual cluster3
 
-    // maps should be populated
+    //We test the ProjectNameToHost Cache before the HostToIp map
+    //This replicates how the proxy accesses these maps as well.
+    //ProjectNameToHost read updates the HostToIP map
```

Comment:
Let's make sure the references to `ProjectNameToHost` and `HostToIp` are accurate at the end, in terms of naming and upper/lower case initials.